arches/app/datatypes/base.py

```diff
@@ -377,6 +377,13 @@ def collects_multiple_values(self):
         """
 
         return False
+    
+    def get_nodevalue_as_list(self, nodevalue):
```

Comment:
Couldn't this method name just be `get_nodevalues`? I think it's generally good to avoid using types in variable names. This method demonstrates why given that it can return either a list or a tuple.

Reply:
fix in latest